tests/Integration/Models/UserTest.php

```
@@ -169,4 +169,48 @@ public function testScopeReturnsOwnTeam(): void
+
+        $this->assertCount($existingTeamIds->count() + $teams->count(), $tenants);
+        $this->assertEqualsCanonicalizing(
+            $existingTeamIds->merge($teams->pluck('id'))->unique()->all(),
+            $tenants->pluck('id')->all()
+        );
+    }
+
+    public function testTeamRolesMorphToManyReturnsAssignedRoles(): void
+    {
+        $user = User::factory()->create();
+        $team = Team::factory()->create();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$team'. (undefined)

(UnusedLocalVariable)